### PR TITLE
Add Authorized pallet

### LINKED ISSUE #292: [PM] Add support for `Authorized` mechanism

### PR DESCRIPTION
**Pull request by c410-f3r**
Depends on #296
Fixes #292

The pallet is not yet integrated into the runtime

### CODE REVIEW

**Comment by c410-f3r** [comment]
On a market resolution, the PM crate will look into this pallet to seek and fetch reported outcomes. Therefore, authorized accounts must register themselves before resolution.

File: zrml/authorized/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    'sp-runtime/std',
+    'zeitgeist-primitives/std',
+    'zrml-market-commons/std'
+]
+
+[package]
+authors = ["Zeitgeist PM <contact@zeitgeist.pm>"]
+edition = "2018"
+name = "zrml-authorized"
+version = "0.1.0"
```
Comment (sea212) [inline zrml/authorized/Cargo.toml:36]: Let's start where all the other pallets are

```suggestion
version = "0.1.2"
```

**Review by sea212 — APPROVED** [review]
Thanks Caio, LGTM.

**Comment by c410-f3r** [comment]
Thanks Harald

**Pull request merged**